electricity/src/bin/electro_collector.rs

```diff
@@ -0,0 +1,51 @@
+use anyhow::{Ok, Result};
+
+use std::env;
+
+use lambda_runtime::{service_fn, LambdaEvent};
+
+use serde_json::Value;
+
+use electricity::db::init_client;
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    tracing_subscriber::fmt()
+        .with_max_level(tracing::Level::INFO)
```

Comment:
At some point, we need to make this setting configurable (using environment variable perhaps).